text/src/autogluon/text/automm/optimization/utils.py

```diff
@@ -462,3 +464,16 @@ def apply_layerwise_lr_decay(
         parameter_group_names[group_name]["params"].append(name)
 
     return list(parameter_group_vars.values())
+
+def mixup(
```

Comment:
Why don't we use timm's mixup class? It supports more features, e.g., cutmix. https://github.com/rwightman/pytorch-image-models/blob/20a1fa63f8ea999dab29d927d5e1866ed3b67348/timm/data/mixup.py#L90